app/views/podcasts/show.rss.builder

```diff
@@ -32,10 +32,10 @@ xml.rss 'xmlns:atom' => 'http://www.w3.org/2005/Atom',
     xml.image do
       xml.url @feed_image.url
       xml.title @feed.title || @podcast.title
-      xml.link @feed_image.link || @podcast.link
+      xml.link @podcast.link
       xml.width @feed_image.width
       xml.height @feed_image.height
-      xml.description @feed_image.description unless @feed_image.description.blank?
+      xml.description @feed.subtitle || @podcast.subtitle
```

Comment:
Again, the docs say:

> <description> contains text that is included in the TITLE attribute of the link formed around the image in the HTML rendering.

I'm not entirely sure what we should be putting there?  But definitely not anything describing the image ... this is a link to the show's website after all.  So I thought... maybe subtitle instead?

In prod, all our image descriptions are null.  So currently not even including this field.

Open to suggestions.